.tekton/push.yaml

```diff
@@ -41,3 +41,28 @@ spec:
     - name: git-auth
       secret:
         secretName: '{{ git_auth_secret }}'
+
+  finally:
```

Comment:
This task needs to go in the `finally` section of the Pipeline, not the PipelineRun.